File: Real_Map/scalingCheck.m
clc
[lt_x, lt_y, lt_utm] = deg2utm(24.7756600,121.0457539);
[lb_x, lb_y, lb_utm] = deg2utm(24.7749817,121.0452239);
[rb_x, rb_y, rb_utm] = deg2utm(24.7741421,121.0463000);
[rt_x, rt_y, rt_utm] = deg2utm(24.7748169,121.0468272);

left = norm([lt_x-lb_x, lt_y-lb_y])
bottom = norm([lb_x-rb_x, lb_y-rb_y])
right = norm([rb_x-rt_x, rb_y-rt_y])
top = norm([rt_x-lt_x, rt_y-lt_y])

File: Real_Map/scalingCheck.py
from pyproj import Proj
import numpy as np

myProj = Proj('+proj=utm +zone=51R, +north +ellps=WGS84 +datum=WGS84 +units=m +nodefs')


lt_x, lt_y = myProj(121.0457539,24.7756600)
lb_x, lb_y = myProj(121.0452239,24.7749817)
rb_x, rb_y = myProj(121.0463000,24.7741421)
rt_x, rt_y = myProj(121.0468272,24.7748169)
lt = np.array((lt_x, lt_y))
lb = np.array((lb_x, lb_y))
rb = np.array((rb_x, rb_y))
rt = np.array((rt_x, rt_y))

print 'left: 	', np.linalg.norm(lt-lb)
print 'bottom:	', np.linalg.norm(lb-rb)
print 'right:	', np.linalg.norm(rb-rt)
print 'top:		', np.linalg.norm(rt-lt)